ios/Classes/FlutterInstanceManagerPlugin.m

```diff
@@ -0,0 +1,20 @@
+#import "FlutterInstanceManagerPlugin.h"
+
+@implementation FlutterInstanceManagerPlugin
++ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
+  FlutterMethodChannel* channel = [FlutterMethodChannel
+      methodChannelWithName:@"flutter_instance_manager"
+            binaryMessenger:[registrar messenger]];
+  FlutterInstanceManagerPlugin* instance = [[FlutterInstanceManagerPlugin alloc] init];
+  [registrar addMethodCallDelegate:instance channel:channel];
```

Comment:
Just like on Android, we don't need method channel communication for the instance_manager plugin so this can be a no-op implementation:

```suggestion
  // no-op
```